backend/stackexchange/stack_api.py

```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
+from stackapi import StackAPI
+from stackapi import StackAPIError
+import json
+from dataclasses import dataclass
+# from dataclasses import make_dataclass
+# from dataclasses_json import dataclass_json
+from dacite import from_dict
+
+
+class ApiResponse:
+    """ Gets the users identified in ids in {ids}.
+    Typically this method will be called to fetch user profiles when you have obtained user ids from some other source, such as /questions.
+    {ids} can contain up to 100 semicolon delimited ids. To find ids programmatically look for user_id on user or shallow_user objects.
+    The sorts accepted by this method operate on the following fields of the user object:
+    reputation – reputation
+    creation – creation_date
+    name – display_name
+    modified – last_modified_date
+    reputation is the default sort.
+    It is possible to create moderately complex queries using sort, min, max, fromdate, and todate.
+    This method returns a list of users. """
+
+    def __init__(self, site: str, user_id: str, order: str = 'desc', sort: str = 'name') -> None:
+        self.site = StackAPI(site)
+        self.order = order
+        self.sort = sort
+        self.api_url = f'users/{user_id}'
+        self.data = self.site.fetch(self.api_url, order=self.order, sort=self.sort)
+
+    def print_api_response_json(self):
+        print(json.dumps(self.data, indent=4))
+
+
+test_user = ApiResponse('stackoverflow', '2335489')
+
+
+# test_user.print_api_response_json()
+
+
+@dataclass
+class UserDataFromJSON:
+    backoff: int
+    has_more: bool
+    page: int
+    quota_max: int
+    quota_remaining: int
+    total: int
+    items: list
+
+    def __post_init__(self):
```

Comment:
We shouldn't fill the fields inside this class. This class should contain the fields only, maybe getters.